common/lib/xmodule/xmodule/split_test_module.py

```diff
@@ -504,11 +513,12 @@ def editable_metadata_fields(self):
 
     @property
     def non_editable_metadata_fields(self):
-        non_editable_fields = super(SplitTestDescriptor, self).non_editable_metadata_fields
+        non_editable_fields = super().non_editable_metadata_fields
         non_editable_fields.extend([
-            SplitTestDescriptor.due,
-            SplitTestDescriptor.user_partitions,
-            SplitTestDescriptor.group_id_to_child,
+            SplitTestBlock.is_entrance_exam,
```

Comment:
This is copied over from the method in the `SequenceDescriptor` class (which is no longer being inherited from).